### ISSUE TITLE
Static build can't handle named path parameter

### ISSUE THREAD

**Issue by thangngoc89** [phenomic/phenomic#1223]
I have a named parameter in a route like this one

```js
<Route
      path="/keywords/:keyword"
      component={createContainer(
        KeywordDetailsRoute.jsComponent,
        KeywordDetailsRoute.queries
      )}
    />
```

It works flawlessly on dev mode build static build can't resolve it.
This is what the folder structure looks like:

![image](https://user-images.githubusercontent.com/3049054/34362265-9bbf1a9e-eaa4-11e7-8110-32313606340a.png)

If this is something that can't be analyzed during static build then we should warns user in dev mode.

**Comment by thangngoc89** [comment]
hmm. After I edited it to use `*` it still can't generate keywords page

**Comment by thangngoc89** [comment]
Actually, this is a limitation / bug ? (cc @MoOx) 
After debugging this, here's what I do to have prerender routes for relations:

Assuming we have a package like this:

```
{
  keywords: ["a", "b", "c"]
}
```

On the route definition, we must use `keywords` as route parameter like this:

```js
<Route
      path="/keywords/:keywords"
      component={KeywordDetailRoute}
    />
```

**Comment by MoOx** [comment]
Indeed, we do not do any magic to resolve a word to its plural version like it was in 0.x (tags => tag, category => categories - we were trying "y => ies" and "+s").

Indeed for meta we need the name (I think). Not sure what is the best to improve the situation.
The * can make sense for ids, but for a named parameter... How do we know the name?

**Comment by thangngoc89** [comment]
I have an idea for this. Workinf on a PoC
